README.md

```diff
@@ -373,6 +375,9 @@ IEEE Int. Symp. Circuits and Systems (ISCAS), Baltimore, MD, 2017.
 - <a name="Haessig17arxiv"></a>Haessig, G., Cassidy, A. Alvarez, R., Benosman, R., Orchard, G.,  
 *[Spiking Optical Flow for Event-based Sensors Using IBM's TrueNorth Neurosynaptic System](https://arxiv.org/pdf/1710.09820.pdf)*,  
 arXiv:1710.09820, 2017.
+- <a name="Zhu2018rss"></a>Zhu, A., Yuan, L., Chaney, K., Daniilidis, K.,  
+*[EV-FlowNet: Self-Supervised Optical Flow Estimation for Event-based Cameras](https://arxiv.org/abs/1802.06898)*,  
```

Comment:
The link on the title of the paper should be the one on the RSS proceedings, once you know the DOI.
The PDF, if available, should go later, before the YouTube link.

Reply:
I've removed the link for now and added a PDF link. Will update with doi once available.